jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantAdapter/VcfFeature.ts

```diff
@@ -0,0 +1,45 @@
+import {default as VCFFeature} from '@jbrowse/plugin-variants/src/VcfTabixAdapter/VcfFeature'
+import { Breakend } from '@jbrowse/plugin-variants/src/VcfTabixAdapter/VcfFeature'
+
+export default class ExtendedVcfFeature extends VCFFeature {
+
+  dataFromVariant(variant: any) {
+    const { REF, ALT, POS, CHROM, INFO, ID } = variant
+    let IMPACT = null
+    let regex = /HIGH/
+    if(INFO["ANN"]){
+        for(let i = 0; i < INFO["ANN"].length; i++){
+            if(/HIGH/g.exec(INFO["ANN"][i])){
+                IMPACT = "HIGH"
+            }
+            else if(IMPACT != "HIGH" && /MODERATE/g.exec(INFO["ANN"][i])){
+                IMPACT = "MODERATE"
+            }
+            else if(IMPACT != "HIGH" && IMPACT != "MODERATE" && /LOW/g.exec(INFO["ANN"][i])){
+                IMPACT = "LOW"
+            }
+        }
+    }
+    INFO["ANN"]["IMPACT"] = IMPACT
```

Comment:
would it be safer to make this a separate top-level property, like INFO['IMPACT']?

Reply:
updated